src/site/content/en/blog/testing-web-design-color-contrast/index.md

```diff
@@ -0,0 +1,433 @@
+everyone. Sometimes testing contrast is easy and sometimes it's really hard. By
+the end of this post you'll have three new tools and techniques for inspecting,
+correcting, and verifying your web design contrast so you can tackle the hardest
+scenarios.
+
+## WCAG and Color Contrast
+
+[W3C’s Web Accessibility Initiative](https://www.w3.org/WAI/) provides
+strategies, standards, and resources to ensure that the internet is accessible
+for as many people as possible. The guidelines that underpin these standards are
+called the Web Content Accessibility Guidelines, or WCAG. The latest stable
+version, [WCAG 2.1](https://www.w3.org/TR/WCAG21/), covers an important
+accessibility requirement: [minimum
+contrast](https://www.w3.org/WAI/standards-guidelines/act/rules/afw4f7/proposed/).
+
+The relationship between two colors in WCAG 2.1 is described by their contrast
+ratio—that is, the number you get when you compare the luminance of two colors.
+Luminance is a way of describing how close a color is to black (0%) or white
+(100%). WCAG defines some rules and calculation algorithms around what that
+contrast ratio needs to be in order for the web to be accessible. There are
+[known issues](https://www.cedc.tools/article.html) with this calculation,
+though. Eventually an even more reliable way will be adopted, but, currently,
+WCAG is the best we have.
+
+### What are the rules?
+
+<div class="table-wrapper scrollbar">
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th>AA</th>
+        <th>AAA</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Body text (< 24px)</td>
+        <td>4.5</td>
+        <td>7</td>
+      </tr>
+      <tr>
+        <td>Large text (> 24px)</td>
+        <td>3</td>
+        <td>4.5</td>
+      </tr>
+      <tr>
+        <td>UI (icons, graphs, etc.)</td>
+        <td>3</td>
+        <td>not defined</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+A higher contrast ratio is scored with a higher number, like 4.5 or 7 instead of
+3. To get more familiar with the scoring table, check out [Polypane's Contrast
+Checker](https://polypane.app/color-contrast/).
+
+
+
+<figure>
+  {% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/WKZJY7co0BADkMfwuqxO.png", alt="Text is shown over purple: one pairing is black text over purple and the other is white text over purple.", width="800", height="438" %}
+  <figcaption>
+    Which of these color pairings do you feel contrast more?
+  </figcaption>
+</figure>
+
+## Testing contrast between colors
+
+So, now that we know what we’re looking for, how do we test this? Here are three
+free tools to assist you in inspecting, correcting and measuring your web site's
+contrast. The strengths and weaknesses of each will be outlined so you can
+confidently test the accessibility of your site’s colors and content in a
+multitude of ways.
+
+1. [Pika](https://superhighfives.com/pika)<br>
+A MacOS system extension, uniquely capable of showing the contrast of any colors
```

Comment:
I'd go with "A MacOS app, uniquely..." as it's not technically a system extension.